Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
No no, it's not bull :hankey: , it should indeed look like that a lot. Things to change from your code snippet : 

- use `array_flip()` to avoid repetition
- throw a deprecation also `$isInverted` is set to `false` (or `true` I don't remember which one is good), because if it is it means that people are not ready to upgrade (I was wrong when saying "when the option is not set")
- make the `NEXT_MAJOR` todo list as small as possible (removing a code block and be finished with the todo list for next major would be best).